knowledge-base/live-traffic-pane.md

```diff
@@ -576,24 +586,15 @@ To select sessions directly related to the currently selected session, use the *
 
 - **Duplicate request**&mdash;selects the session that is identical to the currently selected session; can be accessed with the keyboard by pressing `D`.
 
-### Copying
-
-To place session information in the clipboard, use the **Copy** context menu option, which supports the following options:
-
-- **URL**&mdash;provides the session URL; can be accessed with the keyboard by pressing `Cmd`+`U` (on Mac), or `Ctrl`+`U` (on Windows).
-
-- **Full Summary**&mdash;provides verbose session details; can be accessed with the keyboard by pressing `Cmd`+`C` (on Mac), or `Ctrl`+`C` (on Windows).
-
-- **Copy as Powershell**&mdash;converts the request as a **Windows PowerShell** code and puts it in the system clipboard.
-
-    >important The **Copy as PowerShell** feature generates Windows PowerShell code, not PowerShell (NetCore). These two PS variations use different APIs for HTTP requests (`WebRequest` vs. `HttpClient`), making the generated code compatible only with Windows PowerShell.
+### Marking
 
-- **Copy as Fetch**&mdash;converts the request as a Fetch code and puts it in the system clipboard.
+Use the **Mark** context menu option to apply colors or strike out single or multiple sessions.
 
-- **Copy as Curl**&mdash;converts the request as a Curl command and puts it in the system clipboard.
+1. Select the sessions, open the context menu, and click the **Mark** option.
 
-- **Copy as Python (requests)**&mdash;converts the request as a Python code and puts it in the system clipboard.
+1. Click the preferred marking style to apply it.
```

Comment:
maybe it's just how numbering in this editor is done, but is it supposed to say 1 on both lines here as well?

Reply:
that's fine - it's a markdown thingy to create numbered lists (without actually worrying about the numbers)